src/routes/graph/add-grid.ts

```diff
@@ -4,10 +4,12 @@ import { schema as fieldsToServerFullSchema, FieldsToServerFull } from '../../mo
 
 import { schema as gridSchema, Grid } from '../../models/grid';
 
+// TODO srieding deprecated remove after https://github.com/withthegrid/platform-client/issues/1565
+// scope: entire file.
 interface Request {
   body: {
-    fields: FieldsToServerFull;
-    photo?: string;
+  fields: FieldsToServerFull;
+  photo?: string;
```

Comment:
There's something wrong with the indentation here (also in the other two add-routes)

Reply:
It's fixed